bitnami/kube-prometheus/templates/exporters/kube-controller-manager/service.yaml

```diff
@@ -0,0 +1,25 @@
+{{- if .Values.kubeControllerManager.enabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "kube-prometheus.fullname" . }}-kube-controller-manager
+  namespace: kube-system
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    app: {{ template "kube-prometheus.fullname" . }}-kube-controller-manager
+spec:
+  clusterIP: None
+  ports:
+    - name: http-metrics
+      port: {{ .Values.kubeControllerManager.service.port }}
+      protocol: TCP
+      targetPort: {{ .Values.kubeControllerManager.service.targetPort }}
+{{- if .Values.kubeControllerManager.endpoints }}{{- else }}
+  selector:
+    {{- if .Values.kubeControllerManager.service.selector }}
+{{ toYaml .Values.kubeControllerManager.service.selector | indent 4 }}
+    {{- else}}
+    component: kube-controller-manager
```

Comment:
Review selectors, please

Reply:
what's wrong with the selector?

the controleller-manager PODs have labels:
component=kube-controller-manager,tier=control-plane

do you mean I have to add "tier"?

Reply:
No, I think here we should use the standard `app.kubernetes.io/component`, don't we?

Reply:
Well, this is the `default` without any user based extension, given that the label of the component itself is:
```yaml
Labels:               component=kube-controller-manager
                      tier=control-plane
```

Then for now, until or if k8s decides to change these labels on static pods it should be `component` in my opinion:

https://kubernetes.io/docs/reference/setup-tools/kubeadm/implementation-details/#generate-static-pod-manifests-for-control-plane-components